lib/src/components/NxToast/NxToast.scss

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-toast__wrapper {
+  ///nx-toast__wrapper has position absolute to ensure toasts stay in the top right corner of the page.
+  //Using display:flex and flex-end ensures the right side of the toast is aligned with the right side of 
+  //nx-page-main. The nx-toast_container has position fixed applied so it remains fixed while scrolling.
+  position: absolute;
+  top: 0;
+  right: 0;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: var(--nx-spacing-6x);
+}
+
+.nx-toast__container {
+  position: fixed;
+  width: 500px;
+}
+
+.nx-toast {
+  position: relative;
+  animation: animationIn 0.4s linear;
+
+  &.nx-toast--closing {
+    animation: animationOut 0.4s linear;
```

Comment:
Since the only thing different between here and `.nx-toast` is the `animation-name`, you can just set that property instead of restating the `0.4s linear`.

Reply:
> Since the only thing different between here and `.nx-toast` is the `animation-name`, you can just set that property instead of restating the `0.4s linear`.

Didn't cross my mind, so thanks for cleaning it up. Done here: [ca49163](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/ca49163b4dce7eee38131b2d3ca3b6652eff2c44)